price-oracle/price_oracle.go

```diff
@@ -25,9 +25,10 @@ import (
 )
 
 var (
-	alreadyVotedMapping = make(map[uint64]bool)
-	calculatedDayNumber = make(map[uint64]uint64)
-	priceVotedEventABI  = contractsapi.PriceOracle.Abi.Events["PriceVoted"]
+	alreadyVotedMapping        = make(map[uint64]bool)
```

Comment:
Why don't you just rename the alreadyVoted variable to be more common and reuse it for both cases. They do literally the same with priceUpdatedAlready